src/plugins/input-configurator/public/js/input-configurator.js

```diff
@@ -57,10 +59,18 @@
         self.deletePreset(presetToDelete);
       });
 
-      this.cockpit.on('plugin.inputConfigurator.savePreset', function(presetIn) {
+      this.cockpit.withHistory.on('plugin.inputConfigurator.savePreset', function(presetIn) {
```

Comment:
This seems a bit odd. It implies that the manager when run will scan the message bus to see if anyone had sent a savePreset message and then go ahead and process it.  Usually I think about the lvc pattern as a way to catch up to state changed messages so that things stay in sync... but this seems to be a state change request.